cache/backend_redis.c

```diff
@@ -37,7 +37,7 @@ static int redis_get(redisContext* c, const char* const key, char* res) {
 
   redisReply* reply = redisCommand(c, "GET %s", key);
   if (reply->type == REDIS_REPLY_STRING) {
-    strcpy(res, reply->str);
+    strncpy(res, reply->str, FLEX_TRIT_SIZE_8019);
```

Comment:
FYI. File `hooks/scripts/cpplint.py` searches for the occurrence of `strcpy` and warn.

Reply:
I see. It suggests `snprintf`.